css/main.scss

```diff
@@ -0,0 +1,51 @@
+---
+# Only the main Sass file needs front matter (the dashes are enough)
+---
+@charset "utf-8";
```

Comment:
Yes it does. I was actually having a ton of issues with LESS and grunt in the old version, I had to manually go into the css file (which they recommend not to) and hard code the styling in there.